addons/common/fnc_randomizeFacewear.sqf

```diff
@@ -0,0 +1,44 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_randomizeFacewear
+
+Description:
+    Add config defined weighted random facewear to unit.
+    Uses same config as headgearList[].
```

Comment:
```suggestion
```

Reply:
Need to write somehow how the config looks.

Reply:
```suggestion

CBA_facewearList[] = {}; // default: ignore and use facewear from BIS_fnc_unitHeadgear instead
CBA_facewearList[] = {"", 1}; // no facewear, delete and overwrite facewear from BIS_fnc_unitHeadgear

// 40% no facewear, 30% balaclava, 30% bandana. Weighted randomization: Sum of propabilites must not necessarily equal 1.
CBA_facewearList[] = {"", 0.4, "G_Balaclava_blk", 0.3, "G_Bandanna_blk", 0.3};
```